### ISSUE TITLE
Why most of the clinical notes written earlier than ICU intime, were written over 15 days earlier?

### ISSUE THREAD

**Issue by sudarshan85** [MIT-LCP/mimic-code#555]
Hi,

As part of an analysis, I plotted the number of notes by category that were charted before an ICU intime, relative to that intime. I found that barring Radiology category, most of the notes were written over 15 days earlier than the corresponding ICU intime. [Here](https://github.com/sudarshan85/mimic_extraction/blob/master/icu_admission_pred/notes_analysis.sql) is the SQL script for extracting that information.

I have attached plots to show the distribution of the notes. In particular, I have filtered out erroneous notes and only included notes for which charttime and hadm_id is not null and charttime <= intime. After filtering, there are only 12 categories and about 10% of the total number of notes. The x-axis is in days and 't' is time to ICU admit where t=0 is ICU intime. 
![quad_1](https://user-images.githubusercontent.com/488428/56595507-c5e6b900-65bc-11e9-8c6b-cb0af365f41e.png)
![quad_2](https://user-images.githubusercontent.com/488428/56595508-c5e6b900-65bc-11e9-8207-88910c1f27d9.png)
![quad_3](https://user-images.githubusercontent.com/488428/56595509-c5e6b900-65bc-11e9-96d3-2feaceb91388.png)

My questions are:
1. Why is it that majority of the notes appear 15 days before ICU intime?
2. For notes which have charttime as null, is it possible to impute the charttime from the note text? 

Thanks.

**Comment by alistairewj** [comment]
Looks like this is just a heavy tail. If you increase the bin size I'm sure they are log normally distributed out to some large time.

It is sometimes possible to extract the charttime from the text, e.g. I have done this with echocardiography notes: https://github.com/MIT-LCP/mimic-code/blob/master/concepts/echo-data.sql

**Comment by sudarshan85** [comment]
Thank you @alistairewj 

I took `lop1p` of the periods and plotted the histogram of periods and number of notes. This is what I got:
![log](https://user-images.githubusercontent.com/488428/56615968-52f43700-65ea-11e9-8c10-f66f05e4bd9d.png)

It's interesting that there is a normal distribution there. Moreover, the first 24 hours before ICU intime has the maximum number of notes, and then it begins to go down but reaches a peak at about 11.18 days (expm1(2.5)) before the ICU intime.

Is there any explanation for this distribution of notes before ICU intime or is it just the nature of the data?
